README.md

```diff
@@ -15,5 +34,11 @@ In order to get started you will want to do a couple of things.
 * clone this document repository
 * create a branch with your request ID \(issue number provided when you requested resources\)
 
+## References
+* OpenLab Web Site is hosted on [OpenLab](https://openlabtesting.org/)
+* Join our IRC Channel `#askopenlab` on freenode.net
```

Comment:
s/Channel/channel/

Reply:
Add a link at the end, not everyone is familiar how to connect to freenode [See](https://freenode.net/kb/answer/chat)